guides/source/active_storage_overview.md

```diff
@@ -826,10 +826,11 @@ image_tag file.representation(resize_to_limit: [100, 100])
 
 Will generate an `<img>` tag with the `src` pointing to the
 [`ActiveStorage::Representations::RedirectController`][]. The browser will
-make a request to that controller, which will return a `302` redirect to the
-file on the remote service (or in [proxy mode](#proxy-mode), return the file
-contents). Loading the file lazily allows features like
-[single use URLs](#public-access) to work without slowing down your initial page loads.
+make a request to that controller, which will process the file and upload the
+processed file if necessary, and return a `302` redirect to the file on the
+remote service (or in [proxy mode](#proxy-mode), return the file contents).
```

Comment:
```suggestion
make a request to that controller, which will perform the following:

1. Process the file and upload the processed file if necessary
2. Return an HTTP `302` redirect to the file
  * Either to the remote service (or in [proxy mode](#proxy-mode)
  * Return the file contents
```

I'm not sure this makes sense, but I generally think breaking up long sentences with many states into bullet-points is good practice.

Reply:
Thank you for the feedback! I broke up the sentences, hope it makes it more clear.